tests/test_load.py

```diff
@@ -334,6 +437,63 @@ def test_LocalDatasetModuleFactoryWithoutScript_with_metadata(self):
             for data_file in module_factory_result.builder_kwargs["data_files"]["test"]
         )
 
+    def test_LocalDatasetModuleFactoryWithoutScript_with_one_default_config_in_metadata(self):
+        # cannot parametrize inside unittest.TestCase, don't know how to do it less ugly
```

Comment:
@lhoestq @albertvillanova @mariosasko do you maybe know how to parametrize in classes subclasses from `unittest.TestCase`?